### DISCUSSION TITLE
コミュニケーションを取る場所の案

### DISCUSSION THREAD

**Discussion by Kenzo-Wada** [vike-ja-support/vike-docs-ja#3] [General]
discordがいいのか、discussionで事足りるのか、なにか提案あれば！！

**Comment by usk94** [comment]
ありがとうございます！基本discussionでおこなって、必要であればvike本体のdiscordでDMするのはいかがでしょう？🙋‍♂️

**Reply by Kenzo-Wada** [reply]
了解です！そうしましょう！！（もしもVikeが流行れば、単に日本コミュニティをdiscordでつくる未来があってもよさそうですね！ :ok_man:

**Reply by usk94** [reply]
おお、それはいいアイデアですね👍👍

あと、本業と家庭で、こちらに時間がとれなかったり、返信が遅くなったりすることが多々ある可能性がありますので、スピード感などにストレスに感じられましたら、遠慮なく仰ってください🙇‍♂️🙇‍♂️

**Reply by Kenzo-Wada** [reply]
あ、むしろすみません自分が変な時間に良くcontributeしちゃいがちでした、、、

家庭があるのは皆さんそうですし、給料が発生してるわけでも納期があるわけでもないので、あくまで趣味の範疇で無理なくゆっくり進めていきましょう〜！(むしろ、なにか困ったことあればなんでもおっしゃってください！！)

**Reply by usk94** [reply]
大変助かる言葉をありがとうございます😊
承知しました！それぞれのペースで、コントリビュートしていきましょう👍👍
